CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ set(GKLIB_PATH ${CMAKE_CURRENT_SOURCE_DIR}/GKlib CACHE PATH "path to GKlib")
 option(BUILD_SHARED_LIBS "build a shared library" OFF)
 
 include(${GKLIB_PATH}/GKlibSystem.cmake)
+include(conanbuildinfo.cmake)
```

Comment:
It is good practice to run `conan install` from inside the build-directory so you should handle that too:

```cmake
if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
elseif(EXISTS ${CMAKE_SOURCE_DIR}/conanbuildinfo.cmake)
    include(${CMAKE_SOURCE_DIR}/conanbuildinfo.cmake)
endif()
```